components/WorkoutDetail.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components/native';
+
+export const WorkoutDetail = ({route, navigation}) => {
+  const { workout } = route.params;
+
+  useEffect(()=> {
+    navigation.setOptions({headerShown:false});
+  });
+  
+  return (
+    <Container>
+      <GifImage source={{uri: 'https://media4.giphy.com/media/tn8zWeNYA73G0/200.webp?cid=ecf05e47x13nr03fqr4i6gvj673lsj1w6nim7jgp9ayiopsi&rid=200.webp'}} />
+      <DoTraining>What are you waiting for?</DoTraining>
+      <DoTraining>Just go away {workout.type}!</DoTraining>
+      <BackButton onPress={() => navigation.goBack()}>
+        <TextButton>Go Back!</TextButton>
+      </BackButton>
```

Comment:
Honestly, I didn't understand the images were clickable until I read this row 😅 Is it possible to make it clearer to the user that it's possible to click the advice on the first screen?

Reply:
Also, it would be nice if this screen were scrollable since the content doesn't always fit (on my phone at least) - so the back button gets so far down it's almost not clickable. Try giving the button some bottom padding